.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -160,6 +160,7 @@ jobs:
             --splitting-algorithm least_duration \
             --clean-durations \
             -r fExX \
+            ${{ matrix.python-version == '3.12' && '-m "not udf"' || '' }} \
```

Comment:
We have no Python 3.12 images in production. The udf mark allows us to skip the many tests that rely on server side code execution.